tests/resources/Nimbus-Util.robot

```diff
@@ -306,7 +306,7 @@ Create a Simple VC Cluster
     Set Environment Variable  TEST_DATASTORE  datastore1
     Set Environment Variable  TEST_RESOURCE  ${cluster}
     Set Environment Variable  TEST_TIMEOUT  30m
-    [Return]  ${vc-ip}
+    [Return]  ${esx1}  ${esx2}  ${esx3}  ${vc}  ${vc-ip}
```

Comment:
Yeah I thought about it, looks like other places where this is being called does not return any values. I will make sure if that is the case.